sql/core/src/test/resources/sql-tests/inputs/operators.sql

```diff
@@ -32,6 +32,7 @@ select 1 - 2;
 select 2 * 5;
 select 5 % 3;
 select pmod(-7, 3);
+select cot(1);
```

Comment:
Could you move this to the end of this .sql file? It can reduce the line of code changes.